### ISSUE TITLE
Implement radio sources maps

### ISSUE THREAD

**Issue by zonca** [simonsobs/so_pysm_models#47]
We have radio maps incoming from @xzackli . I imagine the class model for these will be pretty much identical to CIB, but we do need it in so_pysm_models (with a possible additional parameter for flux cut). Thoughts on how to proceed? Attn @marcelo-alvarez

_Originally posted by @msyriac in https://github.com/simonsobs/so_pysm_models/pull/46#issuecomment-535662426_

**Comment by zonca** [comment]
yes, either provide pull request with a similar class to CIB and put maps to NERSC. Or just provide the maps and I'll implement the class.

**Comment by zonca** [comment]
@xzackli @marcelo-alvarez @msyriac do we have any update on radio sources?

**Comment by marcelo-alvarez** [comment]
we have a first version of the radio model ready, @xzackli can you transfer these maps to nersc?

**Comment by zonca** [comment]
@xzackli @marcelo-alvarez any news on this?

**Comment by zonca** [comment]
@xzackli @marcelo-alvarez @msyriac is a map or catalog of these source already available somewhere?

**Comment by xzackli** [comment]
Yes! They are on NERSC at `/global/cfs/cdirs/sobs/www/users/Radio_WebSky/`. They are available via [http](https://portal.nersc.gov/project/sobs/users/Radio_WebSky/), but the official catalogs don't yet have the right permissions for HTTP yet.

**Comment by zonca** [comment]
thanks @xzackli, can you please point me to documentation or add a README.md there that explains how to use those products?

**Comment by zonca** [comment]
thanks @xzackli, can you please point me to documentation or add a README.md there that explains how to use those products?

**Comment by xzackli** [comment]
Working on it!
